libs/domain/picking/src/components/picking/picking.component.ts

```diff
@@ -150,32 +176,50 @@ export class PickingComponent extends PickingListMixin(LitElement) {
                     ></oryx-picking-product-card>
                   `
               )}
+              ${this.renderFinishButton(true)}
             `,
-            () => this.renderFallback()
+            () => this.renderPlaceholderComplete()
           )}
         </div>`
       )}
     `;
   }
 
-  protected renderFallback(): TemplateResult {
-    if (
-      this.pickingList?.items.every(
-        (item) => item.status === ItemsFilters.Picked
-      )
-    ) {
-      return html`
-        <div class="picking-complete">
-          <div class="img-wrap">
-            <oryx-image resource="picking-items-processed"></oryx-image>
-          </div>
-          <h3 class="title-empty">${i18n(`picking.great-job`)}!</h3>
-          <p>${i18n(`picking.all-items-are-processed`)}!</p>
+  protected renderPlaceholderComplete(): TemplateResult {
+    if (this.items?.some((item) => item.status !== ItemsFilters.Picked)) {
+      return html``;
+    }
+
+    return html`
+      <div class="picking-complete">
+        <div class="img-wrap">
+          <oryx-image resource="picking-items-processed"></oryx-image>
         </div>
-      `;
-    } else {
+        <h3 class="title-empty">${i18n(`picking.great-job`)}!</h3>
+        <p>${i18n(`picking.all-items-are-processed`)}!</p>
+      </div>
+      ${this.renderFinishButton()}
+    `;
+  }
+
+  protected renderFinishButton(hasShadow?: boolean): TemplateResult {
+    if (this.items?.some((item) => item.status !== ItemsFilters.Picked)) {
       return html``;
     }
```

Comment:
There is no necessary to return empty templete
```suggestion
  protected renderFinishButton(hasShadow?: boolean): TemplateResult | void {
    if (this.items?.some((item) => item.status !== ItemsFilters.Picked)) {
      return;
    }
```